csharp-intermediate-topics/TuplesInCsharp/TuplesInCsharp/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Tuples in C#");
+
+//CREATING TUPLES USING THE TUPLE CLASS CONSTRUCTOR!!
+
+//Tuple with one member
+var exampleOne = new Tuple<string>("code-maze");
+//Tuple with two members
+var exampleTwo = new Tuple<string, int>("code-maze", 24);
+//Tuple with five members
+var exampleThree = new Tuple<bool, int, string, decimal, string>(true, 3, "CSharp", 23M, "john-doe");
+
+//CREATING TUPLES USING THE TUPLE.CREATE() METHOD
+var example1 = Tuple.Create<string>("code-maze"); //With the specific type indicated
+//var example1 = Tuple.Create("code-maze"); //Same code without type parameter
+
+var example2 = Tuple.Create("code-maze", 24);
+
+var example3 = Tuple.Create(true, 3, "code-maze", 23M, "blockchain");
+
+//Tuple with eight members
+var example4 = Tuple.Create(true, "between", "zero", "and", "one" , false, "bezao", 24);
```

Comment:
Same thing as in the code snippets in the article. Variable names should be more descriptive. I know I'm guilty of doing this myself, but we should strive to do better.